components/renku_data_services/secrets_storage/secret/api.spec.yaml

```
@@ -0,0 +1,230 @@
+openapi: 3.0.2
+info:
+  title: Renku Secrets Storage
+  description: |
+    A service to store and retrieve users' credentials across Renku services.
+  version: 0.1.0
+servers:
+  - url: /api/data
+  - url: /ui-server/api/data
+paths:
+  /secrets/{user_id}:
+    get:
+      summary: Get all user's secrets
+      parameters:
+        - in: path
+          name: user_id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: List of all secrets
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SecretsList"
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - secrets_storage_api
+    post:
+      summary: Add a new secret
```

Here it's missing the `user_id` parameter:
```suggestion
      summary: Add a new secret
      parameters:
        - in: path
          name: user_id
          required: true
          schema:
            type: string
```